spec/domain/queries/find_aggregations_spec.rb

```diff
@@ -0,0 +1,83 @@
+RSpec.describe Queries::FindAggregations do
+  it 'A hash with a Key with each metric name' do
+    resut = Queries::FindAggregations.new.run
+    expect(resut).to be_a(Hash)
+    expect(resut.keys).to eq(Metric.find_all_names)
+  end
+
+  it 'return aggregated values for daily metrics' do
+    edition = create :edition, base_path: '/path/1', date: '2018-01-01'
+    create :metric, edition: edition, date: '2018-01-01', pviews: 2
+    create :metric, edition: edition, date: '2018-01-02', pviews: 3
+
+    result = Queries::FindAggregations.new.run
+    expect(result.fetch('pviews')).to eq(5)
+    expect(result.fetch(:pviews)).to eq(5)
```

Comment:
Feel this is testing 2 different things - that the aggregated value is correct and that we can access the result using a string or symbol? Maybe separate them for clarity?

Reply:
I have finally decided to go with `:symbols` in the response.